flank-scripts/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ shadowJar.apply {
     }
 }
 // <breaking change>.<feature added>.<fix/minor change>
-version = "1.3.1"
+version = "1.3.2"
```

Comment:
feature added -> `1.4.0`

Reply:
hmmm i did. From 1.2 -.13 :P then after each PR it requires a minor change.